WordPress/Classes/System/WordPressAppDelegate.m

```diff
@@ -860,6 +860,21 @@ - (void)listenLocalNotifications
                            selector:@selector(handleLowMemoryWarningNote:)
                                name:UIApplicationDidReceiveMemoryWarningNotification
                              object:nil];
+    
+    [notificationCenter addObserver:self
+                           selector:@selector(create3DTouchShortcutItems)
+                               name:SigninHelpers.WPSigninDidFinishNotification
+                             object:nil];
+    
+    [notificationCenter addObserver:self
+                           selector:@selector(create3DTouchShortcutItems)
+                               name:RecentSitesService.WPTouchedBlogNotification
+                             object:nil];
+    
+    [notificationCenter addObserver:self
+                           selector:@selector(create3DTouchShortcutItems)
+                               name:WPBlogUpdatedNotification
```

Comment:
Great call, verified it doesn't work. Tried something but it may be overkill. Will continue tomorrow.